include/galsim/Silicon.h

```diff
@@ -81,6 +81,7 @@ namespace galsim
     };
 
     int SetOMPThreads(int num_threads);
+    int GetOMPThreads();
```

Comment:
It feels slightly awkward that these live in Silicon.h.  Would they be better placed in, say, Std.h?

Reply:
The declarations can be moved to Std.h, but the instantiations need to live in some .cpp file.  Since Silicon.cpp is the only one using OpenMP at the moment, that seemed the obvious place.  And given that (and that no C++ code ever calls these aside from the pybind11 wrapper), this seems like the right place to put the declarations too.